Jenkinsfile.nightly

```diff
@@ -16,8 +16,6 @@ pipeline {
         stage ("Clean") {
             steps {
                 echo "[INFO] Cleaning the workspace."
```

Comment:
Wondering if this echo should be there?

Reply:
well, the whole stage Clean is then useless, why not remove it at all?

Reply:
removed